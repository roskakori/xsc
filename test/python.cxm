<?xml version="1.0" encoding="utf-8"?>
<text><?cxm python
import errno
eAccess = errno.EACCES
?>errno.EACCES = ${errno.EACCES}; eAccess = ${eAccess}</text>


