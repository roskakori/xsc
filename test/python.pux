<?xml version="1.0" encoding="utf-8"?>
<text><?pux python
import errno
eAccess = errno.EACCES
?>errno.EACCES = ${errno.EACCES}; eAccess = ${eAccess}</text>


