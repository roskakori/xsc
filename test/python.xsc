<?xml version="1.0" encoding="utf-8"?>
<text><?xsc python
import errno
eAccess = errno.EACCES
?>errno.EACCES = ${errno.EACCES}; eAccess = ${eAccess}</text>


