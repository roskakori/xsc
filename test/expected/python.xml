<?xml version="1.0" encoding="utf-8"?><text>errno.EACCES = 13; eAccess =  = 13</text>